packages/react-router/src/components/Router/test/Router.test.tsx

```diff
@@ -40,6 +47,14 @@ describe('Router', () => {
     expect(wrapper).toContainReactComponent(StaticRouter, {location});
   });
 
+  it('mounts a StaticRouter on the server with basename if provided', () => {
+    isClient.mockReturnValue(false);
+    const basename = '/next';
+    const wrapper = mount(<Router basename={basename} />);
```

Comment:
Need to pass location here: https://github.com/Shopify/quilt/pull/1757/checks?check_run_id=1981190998#step:8:3421